docs/04-operation-guides/operations/evnt-01-eventing-monitoring.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Monitoring Kyma Eventing
+---
+
+Kyma Eventing provides several Grafana Dashboards, providing statistics and other information for real-time monitoring of the Eventing backbone.
+
+To access the Grafana Dashboard, we will port-forward the Grafana Service to localhost:
```

Comment:
Does it make sense to mention that grafana will be accessible on http://localhost:8081?

Reply:
added a line